src/transformers/trainer_callback.py

```diff
@@ -60,6 +61,10 @@ class TrainerState:
         best_model_checkpoint (:obj:`str`, `optional`):
             When tracking the best model, the value of the name of the checkpoint for the best model encountered so
             far.
+        early_stopping_patience (:obj:`int`, `optional`):
+            The early stopping patience when evaluating the model during training.
+        early_stopping_patience_counter (:obj:`int`, `optional`):
+            The number of times validation metrics failed to improve.
```

Comment:
Those should not be in the state, just the `EarlyStoppingCallback`.

Reply:
Fixed in https://github.com/huggingface/transformers/pull/8581/commits/b500b6f701163e5bcb97e1c251eb3eac73569aa2